qiskit/circuit/library/standard_gates/equivalence_library.py

```diff
@@ -809,29 +809,34 @@ def _cnot_rxx_decompose(plus_ry: bool = True, plus_rxx: bool = True):
 
 # CSGate
 #
-# q_0: ──■──   q_0: ───────■────────
-#      ┌─┴─┐        ┌───┐┌─┴──┐┌───┐
-# q_1: ┤ S ├ = q_1: ┤ H ├┤ Sx ├┤ H ├
-#      └───┘        └───┘└────┘└───┘
+#                    ┌───┐
+# q_0: ──■──    q_0: ┤ T ├──■───────────■──
+#      ┌─┴─┐         ├───┤┌─┴─┐┌─────┐┌─┴─┐
+# q_1: ┤ S ├ =  q_1: ┤ T ├┤ X ├┤ Tdg ├┤ X ├
+#      └───┘         └───┘└───┘└─────┘└───┘
 q = QuantumRegister(2, "q")
 def_cs = QuantumCircuit(q)
```

Comment:
This change will break the current behavior that we can do
```python
from qiskit import QuantumCircuit, transpile
qc = QuantumCircuit(2)
qc.cs(0, 1)
tqc = transpile(qc, basis_gates=["h", "csx"])
```
Could you instead add your new relation as additional equivalence instead of replacing the old one? The same applies to CSdg below 🙂